### ISSUE TITLE
How to config the nvdla code?

### ISSUE THREAD

**Issue by zhanjf** [redpanda3/soDLA#84]
Hi,
It seems this generator can generate large and small configuration of NVDLA. But how to specify it in sbt run?  
It seems these code can help work

configurations.scala
 nv_small.scala 
 projects.scala
nv_large.scala
odif.scala

BTW: I clone the code and sbt run. there are one error.
soDLA/src/main/scala/nvdla/csb_master/NV_NVDLA_CSB_MASTER_csb2falcon_fifo.scala:401:21: too many arguments (2) for method apply: (clock: chisel3.core.Clock)(block: => T)T in object withClock
[error]   withClock(io.clk, !io.reset_){

I think it should use withClockAndReset in here.

**Comment by redpanda3** [comment]
Hi, you can specify them in projects.scala.

class project_spec extends nv_large/small

**Comment by redpanda3** [comment]
Yes, you are right. CSB is under testing in cocotb, there are some clock
and reset issues...but you are right.
